share/source_mcnp6.F90

```diff
@@ -177,6 +191,17 @@ subroutine source
   pbl%r%tme = 0.0
   pbl%i%ipt = idum(3)
   pbl%i%jsu = 0
-
+  if ((nps .eq. npp) .and. (count_1 + count_2 + count_3 > 0)) then
```

Comment:
What does this output look like for a parallel run? Does it print for each process?

Reply:
Oh, forgot to test it in parallel. Just did it and found ugly output message. Each thread prints the message once. And their numbers are slightly different.

Reply:
Right on. Thanks for looking into it. We might want to do a check that we’re on the master process here and accumulate those stats here too

Reply:
we probably need to do some work to ensure that these variables are accumulated over the difference processes - that might require some assistance from someone with MCNP source code access